unlock-protocol.com/blog/unlock-tokens-launched.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Launching Unlock Discount Tokens
+subTitle: Starting the decentralization of Unlock!
+authorName: Julien Genestoux
+publishDate: December 9, 2020
+description: As an open protocol for memberships, we want Unlock to belong to its users. Today we are introducing the protocol's native token.
+image: /static/images/blog/unlocking-smart-contracts/code.jpeg
+---
+
+Unlock is a protocol. Like another protocol it should not belong to anyone but its community of users and adopters. Today, we're very excited to introduce the Unlock Discount Tokens and start the journey toward a fully decentralized protocol, where users co-own the protocol, as well as earn rewards for their use!
+
+Don't forget to check the step by [step upgrades and deployments as documented on our wiki](https://github.com/unlock-protocol/unlock/wiki/The-Path-to-UDT!)!
+
+# How does UDT work?
+
+The Unlock Discount Token (UDT) is an **optional** ERC20 token that exists to add incentives to actors who are willing to grow and use the Unlock Protocol and network as well as progressively give them control of future upgrades. The smart contract which mints UDT has been deployed to the Ethereum main network and can be found at [`0x90de74265...`](https://etherscan.io/token/0x90de74265a416e1393a450752175aed98fe11517).
+
+Unlock Discount Tokens are minted and granted as rewards upon key purchases to an address designated in the key purchase call, as shown on [this transaction for example](https://etherscan.io/tx/0xb0e5f95ea980c1f096a841e5507f465827411cc29f80f8b95971a5241d9e81bb). Like all of our contracts, it is fully verified on Etherscan and you can access the full [source code on GitHub](https://github.com/unlock-protocol/unlock/)!
+
+We pre-mined 1,000,000 tokens and we'll distribute some of them to early investors and adopters soon, but this pre-mine will also quickly become a minority stake as the supply of tokens will grow as the network grows, since new tokens can be minted for every new key purchase.
+
+As of now, the Unlock and UDT contracts are upgradable using OpenZeppelin and "owned" by a Gnosis multisig wallets whose signers are Unlock employees. Our goal is to soon decentralize that mechanism so that upgrades can only be triggered by UDT holders, after a voting process through a DAO, where votes are weighted by UDT ownership!
+
+Additionaly, the first upgrade that we will then submit will add a mechnism which lets UDT holders claim discounts on their key purchases!
```

Comment:
```suggestion
Additionally, the first upgrade that we will then submit will add a mechanism that lets UDT holders claim discounts on their key purchases!
```

It may be worth seeing if there is a way to highlight this,  it presents a huge benefit to holders but it might be missed

Reply:
Great point!